systems/WikibaseClient/05-Building_Block_View.md

```diff
@@ -3,3 +3,61 @@
 ## Whitebox Overall System
 
 ![Alt Text](./diagrams/05-building-blocks.drawio.svg)
+
+## User Interfaces
+
+## APIs
+
+### Action API Description
+
+![Alt Text](./diagrams/05-api-description.drawio.svg)
+
+|  Building Block | Type/Context  | Responsibility   |
+| ------------ | ------------ | ------------ |
+| DescriptionLookup   | Lookups | Lookup client descriptions from a variety of sources |
+| RepoLinker   | Response | Creates links to Repo Entity concepts |
+
+### Action API EntityUsage
+
+![Alt Text](./diagrams/05-api-entityusage.drawio.svg)
+
+|  Building Block | Type/Context  | Responsibility   |
+| ------------ | ------------ | ------------ |
+| RepoLinker   | Response | Creates links to Repo Entity concepts |
+
+### Action API Meta Wikibase
+
+![Alt Text](./diagrams/05-api-metawikibase.drawio.svg)
+
+|  Building Block | Type/Context  | Responsibility   |
+| ------------ | ------------ | ------------ |
+| ...   | ... | ... |
```

Comment:
```suggestion
| SettingsArray   | Config | WikibaseClient settings containing meta information |
```
or some such? 🤔

Reply:
I'm worried about how we are currently representing access to configuration in different ways.
But perhaps this is again just because things are different in the code right now.
Will fix